Manito/Manito/Screens/CreateRoom/Component/InputDateView.swift

```diff
@@ -19,16 +19,15 @@ class InputDateView: UIView {
         label.font = .font(.regular, ofSize: 18)
         return label
     }()
-    private let dateBackView: UIView = {
-        let view = UIView()
-        view.backgroundColor = .darkGray
-        view.makeBorderLayer(color: .subOrange)
+    private let dateBackView: CalendarView = {
+        let view = CalendarView()
         return view
     }()
-    private var dateLabel: UILabel = {
+    private let dateLabel: UILabel = {
```

Comment:
p5;

```suggestion
    private let dateInfoLabel: UILabel = {
```
뭔가 dateLabel이라고 했을땐 "2022.07.03" 이런 Text가 생각날 것 같아요 !
바로 아래 label.text에 적혀있어서 문제되진 않을 것 같긴한데,
저번에 얘기했을때 모든 문장이나 글들을 한 파일로 분리했을때 조금은 모호할 수 있을 것 같아요 !! 
p10정도되는 p5입니다 ! ^________^

Reply:
냅냅 알겠습니당당

Reply:
[50ff8de](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/97/commits/50ff8ded080450e1ceff51c638baf8be8ebaefbb)